src/CarbonSystemSolver.jl

```diff
@@ -0,0 +1,1750 @@
+module CarbonSystemSolver
+#
+#    Copyright  2013, 2014, 2020, 2021 Guy Munhoven
+#
+#    This file is part of SolveSAPHE v. 2
+#
+#    SolveSAPHE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Lesser General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    SolveSAPHE is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Lesser General Public License for more details.
+#
+#    You should have received a copy of the GNU Lesser General Public License
+#    along with SolveSAPHE.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# ------------------------------------------------------------
+module DissociationConstants
+export DissociationCoefficients
+
+struct DissociationCoefficients
```

Comment:
```suggestion
struct DissociationCoefficients{FT}
```

if you want a type parameter